ppafm/chemistry.py

```diff
@@ -194,7 +195,7 @@ def ringsToMolecule( ring_pos, ring_Rs, Lrange=6.0 ):
 def normalizeSpeciesProbs( species ):
     out = []
     for l in species:
-        renorm=1.0/sum( s[1] for s in l )
+        1.0/sum( s[1] for s in l )
```

Comment:
Does this whole procedure even do something - I do not see any adding to the `out` list in here ...

Reply:
probably not, it seems not to be used anywhere, it is perhaps just remnant of development process